### PR TITLE
Style cop

### PR DESCRIPTION
**Pull request by torbacz**
## Description

Apply StyleCop to several devices:
- ServoMotor
- ShiftRegister
- Sht3x
- Shtc3
- Si7021
- Stopwatch
- System.Device.Model
- Sn74hc595
- Sps30
- SwarmTile
- System.Buffers.Binary.BinaryPrimitives
- System.Drawing
- SSD13xx

## Motivation and Context

- Resolves nanoFramework/Home#1101

## How Has This Been Tested?

- Local build & pipeline build

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by torbacz** [comment]
@josesimoes @Ellerbach 
I know you are busy, but could you review it? I'd rather not wait long time with it, because it could create merge conflicts which may be hard to resolve later on. Thanks!

**Comment by josesimoes** [comment]
Sure thing! Apologies for the delay, please don't think this is because we're dismissing the importance of the PR. 😁

File: devices/Ssd13xx/IFont.cs [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿using System;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
 
 namespace Iot.Device.Ssd13xx
```
Comment (Ellerbach) [inline devices/Ssd13xx/IFont.cs:7]: Note not for this PR: we should rellay have a look at all the IFint using CharacterLcd or extracting something. Will make life easier to many people!

**Review by Ellerbach — APPROVED** [review]
Look all good to me! Thanks for this great work! Really looks really nice!

**Pull request merged**